x-pack/plugins/triggers_actions_ui/public/application/lib/rule_api/map_filters_to_kuery_node.ts

```diff
@@ -83,10 +83,14 @@ export const mapFiltersToKueryNode = ({
   }
 
   if (searchText && searchText !== '') {
+    // if the searchText includes quotes, treat it as an exact match query
+    const value = searchText.match(/["'`](.*?)["'`]/g)
```

Comment:
IDK if it matters but this regex does not check if the opening and closing quotes are matching.
something like `"any text here'` or `"another test'` would match. Maybe `/(["'`])(?:(?=(\\?))\2.)*?\1/g` could be better.

Reply:
Oh true, thanks for catching this! I can change it so that they match

Reply:
Resolved in https://github.com/elastic/kibana/pull/138999/commits/40b6c5b230b85df846a0cf7375df7be6c7ad49ca